test/events/service_contracts/service_contract_cancellation_event_test.rb

```diff
@@ -10,7 +10,7 @@ def test_create
     assert event
     assert_equal event.plan_name, plan.name
     assert_equal event.service_name, service.name
-    assert_equal event.provider, contract.provider_account
+    assert_equal nil, contract.provider_account
```

Comment:
Is it ok that we have events without a provider? Maybe test was boken.

Reply:
I think yes, because I cannot see any such association